<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnapshotPreview</class>
 <widget class="QFrame" name="SnapshotPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kaption - Editor</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <item>
    <widget class="QWidget" name="leftToolbar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="arrowBtn">
        <property name="toolTip">
         <string>Draw an arrow</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/hi16-action-toolbox_arrow.png</normaloff>icons/hi16-action-toolbox_arrow.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boxBtn">
        <property name="toolTip">
         <string>Draw a box</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/hi16-action-toolbox_box.png</normaloff>icons/hi16-action-toolbox_box.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ellipseBtn">
        <property name="toolTip">
         <string>Draw an ellipse</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/hi16-action-toolbox_ellipse.png</normaloff>icons/hi16-action-toolbox_ellipse.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="textBtn">
        <property name="toolTip">
         <string>Write a text</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/hi16-action-toolbox_text.png</normaloff>icons/hi16-action-toolbox_text.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="numberedBtn">
        <property name="toolTip">
         <string>Draw a number in a circle</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/hi16-action-toolbox_number.png</normaloff>icons/hi16-action-toolbox_number.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="centralLayout">
     <item>
      <widget class="SnapshotCanvas" name="snapshotCanvas">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>150</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="bottomToolbar" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,0,0,0">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="filenameLabel">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="filenameLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Image filename</string>
          </property>
          <property name="showClearButton" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="formatCmbBox">
          <item>
           <property name="text">
            <string notr="true">png</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">jpg</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="copyBtn">
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="uploadBtn">
          <property name="text">
           <string>Upload</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveBtn">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelBtn">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="rightToolbar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="propertyToolbar" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="KColorButton" name="colorBtn">
           <property name="toolTip">
            <string>Edit item color</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="color">
            <color>
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </property>
           <property name="defaultColor">
            <color>
             <red>255</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="formatTextBtn">
           <property name="toolTip">
            <string>Edit item font</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../../usr/share/icons/oxygen/16x16/actions/draw-text.png</normaloff>../../../../../usr/share/icons/oxygen/16x16/actions/draw-text.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="sizeWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QSlider" name="widthSlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Edit item size</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>9</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
              <property name="sliderPosition">
               <number>5</number>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBothSides</enum>
              </property>
              <property name="tickInterval">
               <number>2</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>SnapshotCanvas</class>
   <extends>QGraphicsView</extends>
   <header>snapshotcanvas.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
